### PR TITLE
Upgrade websocket library to 1.4.0

### LINKED ISSUE #68: Upgrade java-websocket lib
The library can't be simply updated, since `Base64.decode` method has been removed, but alternative is not easily accessible especially on android.

### PR DESCRIPTION
**Pull request by jimirocks**
Resolves #68

### CODE REVIEW

File: src/main/java/cz/smarteon/loxone/LoxoneCrypto.java [hunk 1]
```diff
@@ -165,9 +167,9 @@ static String decrypt(final String data, final SecretKey sharedKey, final byte[]
             final Cipher cipher = Cipher.getInstance("AES/CBC/PKCS5Padding");
             final IvParameterSpec ivspec = new IvParameterSpec(sharedKeyIv);
             cipher.init(Cipher.DECRYPT_MODE, sharedKey, ivspec);
-            return new String(cipher.doFinal(Base64.decode(data)));
+            return new String(cipher.doFinal(base64ToBytes(data)));
         } catch (NoSuchAlgorithmException | NoSuchPaddingException | BadPaddingException | IllegalBlockSizeException
-                | InvalidAlgorithmParameterException | InvalidKeyException | IOException e) {
+                | InvalidAlgorithmParameterException | InvalidKeyException e) {
```
Comment (PetrZufan) [inline src/main/java/cz/smarteon/loxone/LoxoneCrypto.java]: - remove `IOException` and `Base64` from imports.
- `base64ToBytes()` may throw `IllegalArgumentException` if `data` not in valid base64 format. Don't you want to catch it too?

File: src/main/java/cz/smarteon/loxone/Codec.java [hunk 2]
```diff
@@ -87,6 +88,26 @@ public static String bytesToHex(byte[] bytes) {
...
+    @NotNull
+    public static String bytesToBase64(final @NotNull byte[] bytes) {
+        return Base64.getEncoder().encodeToString(bytes);
+    }
+
+    /**
+     * Decodes given Base64 encoded String to bytes.
+     * @param base64 to decode
+     * @return decoded String
+     */
```
Comment (PetrZufan) [inline src/main/java/cz/smarteon/loxone/Codec.java]: What about to add `@throws IllegalArgumentException`?

File: src/main/java/cz/smarteon/loxone/LoxoneCrypto.java (hunk 1 above)
Comment (jimirocks) [reply src/main/java/cz/smarteon/loxone/LoxoneCrypto.java]: both done

File: src/main/java/cz/smarteon/loxone/Codec.java (hunk 2 above)
Comment (jimirocks) [reply src/main/java/cz/smarteon/loxone/Codec.java]: done

**Pull request merged**